src/com/owncloud/android/ui/activity/Uploader.java

```diff
@@ -363,7 +319,7 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         Vector<OCFile> tmpfiles = getStorageManager().getFolderContent(mFile /*, false*/);
         if (tmpfiles.size() <= 0) return;
         // filter on dirtype
-        Vector<OCFile> files = new Vector<OCFile>();
+        Vector<OCFile> files = new Vector<>();
```

Comment:
Why is the OCFile removed here especially since the variable definition keeps it?